docs/content/stepbystep/IDE-setup.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Setting up the IDE'
+metaTitle: 'IDE Setup | Discordeno'
+metaDescription: 'Setup VSC for discordeno'
+---
+
+# Installing
+
+There are many editors for js/ts, but I recommend Visual Studio Code as its free and fast with a lot of extensions (Including a Deno one)
```

Comment:
- `I` -> `we`

> (Including a Deno one)

- Lowercase `I` here.

- Missing period at the end of the sentence.

Reply:
Why must i change `I` -> `we`? It makes no sense. I recommend VSC, not them